Pipfile

```diff
@@ -20,6 +20,7 @@ requests = "~=2.26"
 geocoder = "~=1.38"
 # For date parsing and manipulation
 python-dateutil = "~=2.8"
+pycryptodome = "*"
```

Comment:
I would pin at least the major version of any production lib we add, out of an abundance of caution.

Reply:
Definitely. Pls pin.

Reply:
Sorry, new to pipenv, I expected it to pin by default. Fixed in 93daa6e.